enhancements/vn-subnet-connectivity/README.md

```
@@ -0,0 +1,525 @@
+
+4. **VM network attachment**: VMs attach only to the subnet's secondary
+   UDN — they do not bind to the pod network. The VM gets its IP from
+   OVN IPAM via DHCP. The default route to the router pod is injected
+   via cloud-init.
+
+### Workflow Description
+
+#### VirtualNetwork Provisioning
+
+1. Tenant creates a VirtualNetwork:
+   ```bash
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)